redash/query_runner/big_query.py

```diff
@@ -242,14 +242,20 @@ def _get_query_result(self, jobs, query):
 
         return data
 
-    def _get_columns_schema(self, table_data):
+    def _get_columns_schema(self, table_data, table_type):
         columns = []
         for column in table_data.get('schema', {}).get('fields', []):
             columns.extend(self._get_columns_schema_column(column))
 
         project_id = self._get_project_id()
         table_name = table_data['id'].replace("%s:" % project_id, "")
-        return {'name': table_name, 'columns': columns}
+        return {
```

Comment:
is this change breaking other data sources or alike (the query auto complete or running queries) ?

Reply:
No, we're adding properties to a dictionary in js(only js reads this), everything except the preview will just ignore them. We can modify every datasource out there to return the same structure, but we want to keep the change to a minimum in case we ever need to move between versions.